lenovo/thinkpad/z/default.nix

```diff
@@ -13,7 +13,6 @@
 
   hardware.enableRedistributableFirmware = lib.mkDefault true;
   hardware.trackpoint.device = lib.mkDefault "TPPS/2 Elan TrackPoint";
-  hardware.video.hidpi.enable = lib.mkDefault true;
```

Comment:
Should we not include `../../common/hidpi.nix` here to not break font sizes in nixos stable?

Reply:
We probably should.

Reply:
Maybe with a comment that it can be dropped after the next release if the console fonts are increased correctly to 32px.

Reply:
https://github.com/NixOS/nixos-hardware/pull/592